doc/man3/OCSP_sendreq_new.pod

```diff
@@ -66,6 +71,12 @@ should be set to B<"/ocspreq">
 
 OCSP_sendreq_nbio() performs I/O on the OCSP request context B<rctx>.
 When the operation is complete it returns the response in B<*presp>.
+It does not support setting a timeout or adding headers and is retained
+for  compatibility; use B<OCSP_sendreq_nbio()> instead.
```

Comment:
This note does not belong here, but to `OCSP_sendreq_bio()`!
Please also remove the double space after "foo".